bower.json

```diff
@@ -8,24 +8,9 @@
   },
   "ignore": [
     "**\/.*",
-    "bower_components",
     "gulp/tasks",
     "node_modules",
     "test"
   ],
-  "license": "Apache 2.0",
-  "dependencies": {
```

Comment:
Does this affect the building of the Bower package? How does Bower know what to include?